custom_components/google_home/models.py

```diff
@@ -69,8 +69,13 @@ def set_timers(self, timers: List[TimerJsonDict]) -> None:
         ]
 
     def get_sorted_alarms(self) -> List[GoogleHomeAlarm]:
-        """Returns alarms in a sorted order"""
-        return sorted(self._alarms, key=lambda k: k.fire_time)
+        """Returns alarms in a sorted order. Inactive alarms are in the end."""
+        return sorted(
+            self._alarms,
+            key=lambda k: k.fire_time
+            if k.status != GoogleHomeAlarmStatus.INACTIVE
+            else k.fire_time + sys.maxsize,
+        )
```

Comment:
oh i really want to start introducing smaller unit tests for features like that to make sure they work 😅

Reply:
you're welcome to add them 😀

Reply:
one day i ll get my hands on them :D